File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/selenium/input-events")

    actions = webdriver.ActionChains(driver)

    print(driver.title)

    actions.click().perform()

    print(driver.find_element(By.CLASS_NAME,"active").text)

    actions.double_click().perform()

    print(driver.find_element(By.CLASS_NAME,"active").text)
    
    actions.context_click().perform()

    print(driver.find_element(By.CLASS_NAME,"active").text)

    driver.close()

File: Selenium/Activities/Activity5.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {

	public static void main(String[] args) {
			
		WebDriver driver =  new FirefoxDriver();
		
		Actions builder = new Actions(driver);
		
		driver.get("https://v1.training-support.net/selenium/input-events");
		
		System.out.println(driver.getTitle());
		
		builder.click().build().perform();
		
		System.out.println(driver.findElement(By.className("active")).getText());
		
		builder.doubleClick().build().perform();
		
		System.out.println(driver.findElement(By.className("active")).getText());
		
		builder.contextClick().build().perform();
		
		System.out.println(driver.findElement(By.className("active")).getText());
		
		driver.close();

	}

}
